tests/unit/lms/services/assignment_test.py

```diff
@@ -116,18 +116,25 @@ def test_upsert_assignment_with_new(
             "custom_ResourceLink.id.history",
         ),
     )
+    @pytest.mark.parametrize(
+        "value,expected",
+        (
+            ("ORIGINAL_ID", "ORIGINAL_ID"),
+            ("ORIGINAL_ID,OLDER_COPY", "ORIGINAL_ID"),
```

Comment:
Original and older here is a bit confusing. Maybe "COPY_OF_ORIGINAL_ID,ORIGINAL_ID"?